examples/rethinkdb/image/run.sh

```diff
@@ -16,14 +16,19 @@
 
 set -o pipefail
 
+# TODO: Switch this example to get the IP address from DNS instead of using the
+# kubernetes API.
+
 IP=""
-if [[ -n "${KUBERNETES_RO_SERVICE_HOST}" ]]; then
+if [[ -n "${KUBERNETES_SERVICE_HOST}" ]]; then
 
-  : ${NAMESPACE:=rethinkdb}
+  : ${NAMESPACE:=rethinkdb} # TODO: get namespace from downward API
   # try to pick up first different ip from endpoints
   MYHOST=$(ip addr | grep 'state UP' -A2 | tail -n1 | awk '{print $2}' | cut -f1  -d'/')
-  URL="${KUBERNETES_RO_SERVICE_HOST}/api/v1beta3/namespaces/${NAMESPACE}/endpoints/rethinkdb-driver"
-  IP=$(curl -s ${URL} | jq -s -r --arg h "${MYHOST}" '.[0].subsets | .[].addresses | [ .[].IP ] | map(select(. != $h)) | .[0]') || exit 1
+  URL="https://${KUBERNETES_SERVICE_HOST}${KUBERNETES_SERVICE_PORT}/api/v1beta3/namespaces/${NAMESPACE}/endpoints/rethinkdb-driver"
```

Comment:
Should be  (note the **:**)
`URL="https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}/api/v1beta3/namespaces/${NAMESPACE}/endpoints/rethinkdb-driver"`
right?  Not
`URL="https://${KUBERNETES_SERVICE_HOST}${KUBERNETES_SERVICE_PORT}/api/v1beta3/namespaces/${NAMESPACE}/endpoints/rethinkdb-driver"`